modules/core/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 project(node_rapids_core VERSION 0.0.1 LANGUAGES C CXX)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+unset(CMAKE_LIBRARY_OUTPUT_DIRECTORY)
```

Comment:
I am curious on why you need to unset the location of libraries

Reply:
Oh this is to work around an issue with the command crafted by `cmake-js` -- they explicitly pass `-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=./build/Release`.

Normally this would be fine, but it causes CMake to put `libcudf.so` into the build dir of the first project to build it, instead of the common shared dir (i.e. `modules/.cache/build/cudf-build/libcudf.so`).

This causes subsequent clean-builds to fail in the following way:

1. `cmake-js` deletes `modules/cudf/build/Release` (`node_cudf` is the first project to build `libcudf.so`)
2. `cudf_DIR` is set to the common build dir
   * `CPMFindPackage(cudf)` sees `cudf-config.cmake` etc. and thinks `libcudf` has been "installed"
4. But `libcudf.so` was in `node_cudf`'s build dir, so it fails with a "missing libcudf.so and no rule to make it" error